packages/runtime/test/src/runtime.ts

```diff
@@ -992,6 +992,26 @@ describe("Stateful Smart Contracts", function () {
 			RUNTIME_ERRORS.TEAL.PROGRAM_VERSION_MISMATCH
 		);
 	});
+
+	it("Should throw and error when local/global schema entries exceeds the limit (AppDeploymentFlags)", () => {
```

Comment:
Add one more test when global schema bound of limit.

Reply:
Done